.travis.yml

```diff
@@ -50,8 +50,7 @@ env:
   matrix:
     - NGINX_VERSION=1.15.8 OPENSSL_VER=1.0.2q OPENSSL_OPT="" OPENSSL_PATCH_VER=1.0.2h
     - NGINX_VERSION=1.15.8 OPENSSL_VER=1.1.0j OPENSSL_OPT="" OPENSSL_PATCH_VER=1.1.0d
-    # TODO: when adding an OpenSSL version >= 1.1.1, please add "enable-tls1_3"
-    # to $OPENSSL_OPT.
+    - NGINX_VERSION=1.15.8 OPENSSL_VER=1.1.1b OPENSSL_OPT="enable-tls1_3" OPENSSL_PATCH_VER=""
```

Comment:
- Could we bump to 1.1.1c?
- I saw the TODO note on this diff, but nonetheless, TLS 1.3 is enabled by default in OpenSSL 1.1.1 afaik. Do we really need the `enable-tls1_3` option?